### ISSUE TITLE
NOT SUPPORTED: keyword "id", use "$id" for schema ID

### ISSUE THREAD

**Issue by FiretronP75** [angular-eslint/angular-eslint#1369]
Got this error from following the one simple instruction in the README.

```
ng add @angular-eslint/schematics
ℹ Using package manager: yarn
✔ Found compatible package version: @angular-eslint/schematics@1.0.0.
✔ Package information loaded.

The package @angular-eslint/schematics@1.0.0 will be installed and executed.
Would you like to proceed? Yes
✔ Packages successfully installed.
NOT SUPPORTED: keyword "id", use "$id" for schema ID
error Command failed with exit code 1.
```
Angular CLI version 14

**Comment by JamesHenry** [comment]
This suggests something stale in a cache on your machine somewhere: Being explicit with the version should help you here (and I recommend it in general).

```sh
ng add @angular-eslint/schematics@14
```

**Comment by FiretronP75** [comment]
Yes, that fixed it. Should probably update the readme with that.
